platform/viewer/src/appInit.js

```diff
@@ -49,6 +50,14 @@ function appInit(appConfigOrFunc, defaultExtensions) {
     appConfig,
   });
 
+  if (window) {
```

Comment:
This stuff loads the theme configuration values at startup, defaulting to the theme.json5 is not otherwise specified.